https://github.com/ntop/nDPI/pull/662

From e91123e17a6ebe2cb1f718aa3e44edb10b707779 Mon Sep 17 00:00:00 2001
From: "Bernhard M. Wiedemann" <bwiedemann@suse.de>
Date: Thu, 24 Jan 2019 14:21:06 +0100
Subject: [PATCH] Use ChangeLog date instead of build date

in order to make builds reproducible.
See https://reproducible-builds.org/ for why this is good.

This date call works with GNU date and BSD date.
Also use UTC/gmtime to be independent of timezone.
---
 configure.seed | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.seed b/configure.seed
index 1aa68f17..006e6d97 100644
--- a/configure.seed
+++ b/configure.seed
@@ -30,7 +30,7 @@ if test -d ".git"; then :
      GIT_RELEASE="${PACKAGE_VERSION}-${GIT_NUM}-${GIT_TAG}"
 else
      GIT_RELEASE="${PACKAGE_VERSION}"
-     GIT_DATE=`date`
+     GIT_DATE=`date -u -r CHANGELOG.md`
 fi
 
 AC_DEFINE_UNQUOTED(NDPI_GIT_RELEASE, "${GIT_RELEASE}", [GIT Release])
